README.md

```diff
@@ -1,3 +1,10 @@
+This fork add support for: `aNy-CasE` or `Any cAse` - always converted to snake_case
+Also I added support for new: `Title Case`  
+
+<video controls="true" allowfullscreen="true" poster="path/to/poster_image.png">
+    <source src="https://user-images.githubusercontent.com/13521338/113520457-8213d100-9593-11eb-974b-b8172eddd950.mp4" type="video/mp4">
+</video>
+
```

Comment:
Hey! thanks for your contribution!

Could you please make this look like it is part of the same repo? 🙇